stories/checkbox.stories.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+import Checkbox from 'GoogleComponents/checkbox';
+
+storiesOf( 'Global', module )
```

Comment:
Not sure if we want to include new stories in this PR that aren't related to `classnames` use. Did you check if there were any components in `stories` which need to be updated to use `classnames` as well?

Make sure that any usage there is updated but let's hold new stories for a dedicated PR unless the PR touches relevant components.